### DISCUSSION TITLE
[Plugin]: CROWDIN SUPPORT !

### DISCUSSION THREAD

**Discussion by Mara-Li** [ObsidianPublisher/obsidian-github-publisher#292] [Announcements]
Hello!
You can now translate the plugin with crowdin: https://crowdin.com/project/obsidian-github-publisher

I added the support of all 30 (minus english) of popular language, including chinese, russian, or italian! It will now easier to translate the plugin into YOUR language, and help other to discover GitHub Publisher :D

**Comment by comradekingu** [comment]
I think you want Weblate instead of Crowdin. :)

**Reply by Mara-Li** [reply]
What are the difference?:)
